lib/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounter.scss

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use "sass:math";
+@use "../../scss-shared/nx-text-helpers";
+
+@mixin counter-colors {
+  @each $severity in critical, high, medium, low, none {
+    &--#{$severity} {
+      background-color: var(--nx-color-vulnerability-#{$severity});
+      color: var(--nx-color-vulnerability-#{$severity}-text);
+      border: 1px solid var(--nx-color-vulnerability-#{$severity}-border);
+    }
+  }
+}
+
+.nx-small-vulnerability-counter {
+  $height: 25px;
```

Comment:
Could a shared variable be created to ensure both small counters keep the same height?
https://github.com/sonatype/sonatype-react-shared-components/blob/da283bc73482926ab19e666cf56ea54115af8187/lib/src/components/NxSmallThreatCounter/NxSmallThreatCounter.scss#L20

Reply:
I created new variables in d3bce8bc1855082ce4163e21d0a2367b2457e319, let me know if it that implementation is ok

Reply:
Hmm, I think we want to avoid changes to Threat components unless we abstract this further.
I don't think we want to use a variable to tie these two together though.

Reply:
> Hmm, I think we want to avoid changes to Threat components unless we abstract this further. We can create a base style thats common across these two. I don't think we want to use a single variable to tie these two properties though.

One of the reasons to create this components for vulnerabilities is to be able to use it together with the policy violation threat level equivalent side to side (or at least across pages and tiles) in SBOM Manager.

The new Applications page is an example of that, so I’d expect both components to have the same height for consistency and to avoid giving a false impression that one is more important than the other.

Reply:
> > Hmm, I think we want to avoid changes to Threat components unless we abstract this further. We can create a base style thats common across these two. I don't think we want to use a single variable to tie these two properties though.
> 
> One of the reasons to create this components for vulnerabilities is to be able to use it together with the policy violation threat level equivalent side to side (or at least across pages and tiles) in SBOM Manager.
> 
> The new Applications page is an example of that, so I’d expect both components to have the same height for consistency and to avoid giving a false impression that one is more important than the other.

Yes but the height property is not the only thing defining the dimension.
That's why the ideal approach if we want to do this we have to abstract a common base style which includes the height not just a single variable.

Reply:
> Yes but the height property is not the only thing defining the final dimension (height) of the component. That's why the ideal approach if we want to do this we have to abstract a common base style which includes the height (not just a single height variable, which in my opinion wouldn't be useful at all and will just be an outlier one off coupling).
> 
> We discussed about doing this for all the new vuln components and their corresponding threat components but it was decided for the first iteration we would have duplicate implementation since we need these components soon and there may be design further changes down the road based on feedback and learnings from this first iteration.
> 
> But if it's insisted we can spend some time to do that work.

Ah, thanks for the context!
It can be left as it was originally developed then

Reply:
Reverted changes on [987ad66](https://github.com/sonatype/sonatype-react-shared-components/pull/1200/commits/987ad662a8f2bca6b01578f882d1b64cdac6eb3b)